doc/api/net.md

```diff
@@ -625,15 +625,16 @@ initialDelay will leave the value unchanged from the default
 
 Returns `socket`.
 
-### socket.setNoDelay([noDelay])
+### socket.setNoDelay([enable])
 <!-- YAML
 added: v0.1.90
 -->
 
-Disables the Nagle algorithm. By default TCP connections use the Nagle
-algorithm, they buffer data before sending it off. Setting `true` for
-`noDelay` will immediately fire off data each time `socket.write()` is called.
-`noDelay` defaults to `true`.
+Sets `noDelay` option, which disables the Nagle algorithm. By default TCP
+connections use the Nagle algorithm, they buffer data before sending it off.
+Setting `true` for `noDelay` will immediately fire off data each time
+`socket.write()` is called.
+`enable` defaults to `true`.
```

Comment:
Another way this could be reworded is:

```
By default, TCP connections use the Nagle algorithm to buffer data before
sending. Calling `socket.setNoDelay(true)` disables such buffering so that
data is sent each time `socket.write()` is called.

The `enable` argument defaults to `true`.
```